src/ray/raylet/scheduling/local_resource_manager.h

```diff
@@ -26,18 +26,17 @@
 #include "ray/gcs/gcs_client/accessor.h"
 #include "ray/gcs/gcs_client/gcs_client.h"
 #include "ray/raylet/scheduling/cluster_resource_data.h"
-#include "ray/raylet/scheduling/cluster_resource_scheduler_interface.h"
 #include "ray/raylet/scheduling/fixed_point.h"
-#include "ray/raylet/scheduling/scheduling_ids.h"
-#include "ray/raylet/scheduling/scheduling_policy.h"
 #include "ray/util/logging.h"
 #include "src/ray/protobuf/gcs.pb.h"
 
 namespace ray {
 
-using rpc::HeartbeatTableData;
-
 /// Class manages the resources of the local node.
+/// It is responsible for allocating/deallocating resources for (task) resource request;
+/// it also supports creating a new resource or delete an existing resource.
+/// Whenever the resouce changes, it notifies the subscriber of the change.
+/// This class is not thread safe.
```

Comment:
Could we add which thread it is expected to be running in? like which event loop.

Reply:
We can revisit this once we finished the cluster resource manager part. I'm not sure if this class should be aware of the event loop.. yet.